### DISCUSSION TITLE
Achievement EventCommands

### DISCUSSION THREAD

**Discussion by AnimeDrache** [Zrips/CMI#8014] [Enhancement]
I can start events for players who leave, go afk etc but not for achievement. Would be cool to play a Sound when someone reaches an achievement. So I would need that special EventCommand in the EventCommands.yml

**Comment by Slarof** [comment]
Yes, I think it is necessary :)

For example: I need virtual money (for /pay, etc.) to be issued when each achievement is completed

It is desirable that this also be divided into varieties of achievements :3

**Comment by AnimeDrache** [comment]
Sounds like a cool idea so yes we need it hehe =P

**Comment by Zrips** [comment]
Was did looked into this, somewhat issue is that advancement doesn't have pretty looking names, so what we can have is a return value like "recipes/building_blocks/emerald_block" for advancement name/identificator. Additionally advancement event gets fired for each one of them, so some actions can trigger this action like 10 times. But well, will add with next update and you can mess around with it.

**Comment by AnimeDrache** [comment]
> Was did looked into this, somewhat issue is that advancement doesn't have pretty looking names, so what we can have is a return value like "recipes/building_blocks/emerald_block" for advancement name/identificator. Additionally advancement event gets fired for each one of them, so some actions can trigger this action like 10 times. But well, will add with next update and you can mess around with it.

I am not fully sure what you mean. Like couldnt we use the "EventCommands.yml" to let a message appear or a command trigger when someone did an Achievement? =)
